packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -96,7 +97,13 @@ class CodeSizeController {
     changeSnapshotFile(pathToFile);
 
     // Build a [Map] object containing heirarchical information for [inputJsonMap].
-    final processedJsonMap = treemapFromJson(_jsonForFile(pathToFile));
+    Map<String, dynamic> processedJsonMap;
+    if (pathToFile.contains('app_size')) {
```

Comment:
the call to _jsonForFile is used in both branches. Move it outside of the if.
e.g.
`final json = _jsonForFile(pathToFile);`

Reply:
Done